### ISSUE TITLE
Setting a theme throws "UnicodeDecodeError"

### ISSUE THREAD

**Issue by tastyminerals** [dunovank/jupyter-themes#231]
The error is thrown in anaconda virtual environment.

```shell
$ pip install jupyterthemes
```
```
$ jt -l
Available Themes: 
   chesterish
   grade3
   gruvboxd
   gruvboxl
   monokai
   oceans16
   onedork
   solarizedd
   solarizedl
```
```
$ jt -t onedork
Traceback (most recent call last):
  File "/home/letian/anaconda3/envs/tfdeeplearning/bin/jt", line 11, in <module>
    sys.exit(main())
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/jupyterthemes/__init__.py", line 305, in main
    dfonts=args.defaultfonts)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/jupyterthemes/__init__.py", line 101, in install_theme
    style_css = stylefx.less_to_css(style_less)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/jupyterthemes/stylefx.py", line 63, in less_to_css
    style_css = lesscpy.compile(tempfile)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/lesscpy/__init__.py", line 18, in compile
    p.parse(file=file)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/lesscpy/lessc/parser.py", line 154, in parse
    file, lexer=self.lex, debug=debuglevel)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/ply/yacc.py", line 331, in parse
    return self.parseopt_notrack(input, lexer, debug, tracking, tokenfunc)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/ply/yacc.py", line 1024, in parseopt_notrack
    lexer.input(input)
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/site-packages/lesscpy/lessc/lexer.py", line 475, in input
    self.lexer.input(f.read())
  File "/home/letian/anaconda3/envs/tfdeeplearning/lib/python3.5/encodings/ascii.py", line 26, in decode
    return codecs.ascii_decode(input, self.errors)[0]
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 42624: ordinal not in range(128)
```

**Comment by dunovank** [comment]
Would you mind upgrading to version 0.18.8 to see if this is fixed? And could you report your results back here? Thanks!  

```sh
pip install --upgrade jupyterthemes
```

**Comment by dunovank** [comment]
Has been fixed in v0.18.9 (see duplicate issues #217 and #220 )
